### ISSUE TITLE
each question in v2.q exists twice, and the original question has been lost

### ISSUE THREAD

**Issue by freefuiiismyname** [ankit-ai/SQUAD2.Q-Augmented-Dataset#1]
Thank you for your work!
When I analyzed the data, I found that each question in v2.q existed twice, and the original question had been lost.Maybe your code need a 'deepcopy' function, to ensure original question not be covered.

**Comment by freefuiiismyname** [comment]
And i tried v2.q, v2q50 with BERT-base, the score of dev set lower about 1 point than raw dataset

**Comment by akfuster** [comment]
Yeah it looks like he's copying the dict in line 51 of augment.py. Should instead use a deepcopy.
